lib/services/local/users.go

```diff
@@ -62,8 +62,8 @@ var GlobalSessionDataMaxEntries = 5000 // arbitrary
 type IdentityService struct {
 	backend.Backend
 	log logrus.FieldLogger
-	// ServiceClock is used in tests to desync the service clock from the backend clock.
-	ServiceClock clockwork.Clock
+	// clock is used in tests to desync the service clock from the backend clock.
+	clock clockwork.Clock
```

Comment:
```suggestion
	desyncedClockForTesting clockwork.Clock
```

So it's painfully clear?